app/main.py

```diff
@@ -1,8 +1,48 @@
+from typing import List
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+                 ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one

Reply:
I don't understand how it's must be, because when i write all arguments on new line i have problems with flake8:
E126 continuation line over-indented for hanging indent
E126 continuation line over-indented for hanging indent
E126 continuation line over-indented for hanging indent
E123 closing bracket does not match indentation of opening bracket's line

Reply:
sample above

Reply:
Done